### DISCUSSION TITLE
Minio Tiering and minimum lifecycle rule size to maintain metadata on tier 1.

### DISCUSSION THREAD

**Discussion by tdsn0007** [minio/minio#20685] [Q&A]
Hi everyone!

I have a customer with a minio cluster with a 16 nodes with tiering using Veeam. 1,3Pb of space on tier 2 and only 84Tb on tier 1.

We have this rule to tiering in all my buckets:

mcli ilm rule add --transition-days 0 --transition-tier ONEZONE_IA --size-gt 8KiB s3sp0/BUCKET

Now we have a 1.2B of objects on all tiers and "only" 1B of objects on tier 2. We have 200M of objects on tier 1. And on tier 1 we have 50% of free space available now.

**Tier 1 is only needed to metadata, not files.**

My question:

**Is possible to move all objects with exception of metadata to tier 2?** For example:

mcli ilm rule add --transition-days 0 --transition-tier ONEZONE_IA -**-size-gt 128BiB** s3sp0/BUCKET

I don´t know how much space metadata is using, but i think is not 35TB of actual T1 use. 

Thanks for your help.

**Comment by klauspost** [comment]
Ask your customer to get a [support subscription](https://min.io/pricing). We can discuss these questions in that context.

**Comment by tdsn0007** [comment]
I already asked about this. But the pricing started to be much expensive for this customer. Unfortunately.

About the tiering, documentation is not clear. 

Metadata remains in tier 1 even when we place very small files (bytes) to be migrated to tier 2?

Is just my question.

**Reply by klauspost** [reply]
Yes, Metadata always remains. That is by design.

**Reply by tdsn0007** [reply]
Thanks!

Is possible to know what is the size of metadata in object storage?
